crates/lib/src/compiler/isa/mod.rs

```diff
@@ -64,8 +64,13 @@ impl TryFrom<InstructionSetArchitecture> for Compiler {
         let qubits = qubits
             .into_iter()
             .map(|(k, v)| (k.to_string(), v))
+            .filter(|(_, q)| q.has_valid_operations())
+            .collect();
+        let edges = edges
+            .into_iter()
+            .map(|(k, v)| (k.to_string(), v))
+            .filter(|(_, e)| e.has_valid_operations())
             .collect();
```

Comment:
To get ahead of a related error - should only be those edges for which both qubits are also included